src/Test/Utilities/DisposableFile.cs

```diff
@@ -51,5 +53,31 @@ public void Dispose()
                 }
             }
         }
+
+        [DllImport("kernel32.dll", PreserveSig = false)]
```

Comment:
Not sure if this is new code, but "kernel32.dll" is not portable. By default, you should be using API sets: https://msdn.microsoft.com/en-us/library/windows/desktop/hh802935(v=vs.85).aspx, however, you have a problem. Windows 7 didn't support them. For CoreCLR we solve this by shipping them downlevel, but .NET Framework does not do this. To be portable, you'll basically need to look for the API set, then fall back to "kernel32". You'll start seeing issues on things like Nano server if you don't do this.